docs/basic-docs/atomicdex/atomicdex-api.md

```diff
@@ -725,19 +725,33 @@ Set the value of the `mm2` parameter in the [coins](../../../basic-docs/atomicde
 
 For terminal interface examples, see the examples section below.
 
+#### Using AtomicDEX Software on an QRC20-Based Network
+
+The following information can assist the user/developer in connecting AtomicDEX software to the QRC20-based network:
+
+- Swap smart contract on the QRC20 mainnet: [0x2f754733acd6d753731c00fee32cb484551cc15d](https://qtum.info/contract/2f754733acd6d753731c00fee32cb484551cc15d)
+  - Main-net nodes maintained by the Komodo team: <b>http://electrum1.cipig.net:10050</b>, <b>http://electrum2.cipig.net:10050</b>, <b>http://electrum3.cipig.net:10050</b>, <b>http://electrum1.cipig.net:20050</b>, <b>http://electrum2.cipig.net:20050</b>, <b>http://electrum3.cipig.net:20050</b>
```

Comment:
Electrums do not support HTTP protocol. Please change it to `TCP` and `SSL` accordingly.

Reply:
Thank you, I missed that when I copied block from ETH `<b>http://eth1.cipig.net:8555</b>`

Reply:
Please note I decided to remove Qtum electrums from doc, because electrums of other UTXO-based coins are not mentioned in this doc. The info can be found in `coins` repo